js/admin/src/panes/ProfileConfigurePane.js

```diff
@@ -117,12 +117,27 @@ export default class ProfileConfigurePane extends Component {
         }, [
             m('legend', [
                 exists ? m('div', {className: 'ButtonGroup pull-right'}, [
+					Button.component({
+                            className: 'Button Button--icon',
+                            icon: "pencil-square-o",
+                            onclick: function onclick(e) {
+                                return _this4.toggleField(e);
```

Comment:
what the hell are these `_this4`, did you copy something from `extension.js` ? Also the method call does not adhere to our usual ES6 format